### ISSUE TITLE
** ERROR ** Configuration profile name "CloudConnexa" already exists

### ISSUE THREAD

**Issue by deanfourie1** [OpenVPN/openvpn3-linux#236]
I'm trying to re deploy a new connector profile from CloudConnexa to a linux client, I cannot for the life of me get it to deploy once a previous profile has been installed.

Installing via the provided script debian_12.sh

How can I remove the previous profiles? Profiles do not show under openvpn3 configs-list

Thanks

**Comment by deanfourie1** [comment]
> OpenVPN 3 Linux is "user aware". So if the profile was imported as `root` and you run `openvpn3 configs-list` as another user, then it won't show up _unless_ that user is granted access to this profile (via `openvpn3 config-acl`).
> 
> I would try to run `openvpn3 configs-list` as `root` and see what that shows.

Yes thank you, that worked! I now see the profile listed as CloudConnexa.

Could you tell me how to deleted it? As I do not know the config path?

Thanks

**Comment by deanfourie1** [comment]
When I attempt 

openvpn3 config-remove --config CloudConnexa

I get the following error

** ERROR ** more then one configuration profile was found with the given name.

**Comment by dsommers** [comment]
Use `openvpn3 config-lists -v` ... and then `openvpn3 config-remove --config-path ....`

**Comment by deanfourie1** [comment]
Sorry but I do not know the config path, the script installed it automatically.

**Comment by deanfourie1** [comment]
Oh sorry gotcha! -v gave me what I needed.

Let me try this

**Comment by deanfourie1** [comment]
Thank you very much, this has worked! New profile has been installed and connector online.

FYI, I think the documentation really lacks this information.

**Comment by dsommers** [comment]
Which information do you mean?  Cloud Connexa or the man pages in OpenVPN 3 Linux?

https://codeberg.org/OpenVPN/openvpn3-linux/src/branch/master/docs/man/openvpn3-configs-list.1.rst
https://codeberg.org/OpenVPN/openvpn3-linux/src/branch/master/docs/man/openvpn3-config-remove.1.rst
https://codeberg.org/OpenVPN/openvpn3-linux/src/branch/master/docs/man/openvpn3-config-acl.1.rst

Just to be clear, I'm very much in favour of improving documentation .... no matter which one :slightly_smiling_face:

**Comment by dsommers** [comment]
I just remembered that the `openvpn-connector-setup` command does have a `--force` option too ...

```
$ openvpn-connector-setup --help
usage: openvpn-connector-setup [options]

OpenVPN Connector Setup utility

optional arguments:
  -h, --help            show this help message and exit
  --mode MODE           Defines how configuration profiles are imported and
                        stored (default: systemd-unit)
  --token TOKEN_VALUE   This value is provided by the CloudConnexa web portal.
  --name NAME           Configuration profile name to use. Default:
                        "CloudConnexa"
  --force               Overwrite configuration profile if it already exists
  --autoload-file-prefix AUTOLOAD_FILE_PREFIX
                        Configuration filename to use. Default: connector.conf
  --no-start            Do not start and configure the profile to start at
                        boot
  --dco                 Use OpenVPN Data Channel Offload (DCO) by default
  --version             Show openvpn-connector-setup version
```
